okhttp/src/main/java/okhttp3/CipherSuite.java

```diff
@@ -398,7 +398,7 @@
   public static final CipherSuite TLS_AES_256_GCM_SHA384 = init("TLS_AES_256_GCM_SHA384", 0x1302);
   public static final CipherSuite TLS_CHACHA20_POLY1305_SHA256 = init("TLS_CHACHA20_POLY1305_SHA256", 0x1303);
   public static final CipherSuite TLS_AES_128_CCM_SHA256 = init("TLS_AES_128_CCM_SHA256", 0x1304);
-  public static final CipherSuite TLS_AES_256_CCM_8_SHA256 = init("TLS_AES_256_CCM_8_SHA256", 0x1305);
+  public static final CipherSuite TLS_AES_128_CCM_8_SHA256 = init("TLS_AES_128_CCM_8_SHA256", 0x1305);
```

Comment:
This is a backwards-incompatible API change. But it would never have worked previously.

Reply:
https://tools.ietf.org/html/draft-ietf-tls-tls13-28#appendix-B.4

Reply:
https://tools.ietf.org/html/rfc8446#appendix-B.4